deps.qt/qt6.ps1

```diff
@@ -1,132 +1,281 @@
+    $Options += @(
+        '-DFEATURE_androiddeployqt:BOOL=OFF'
+        '-DFEATURE_brotli:BOOL=OFF'
+        '-DFEATURE_cups:BOOL=OFF'
+        '-DFEATURE_dbus:BOOL=OFF'
+        '-DFEATURE_doubleconversion:BOOL=ON'
+        '-DFEATURE_freetype:BOOL=OFF'
+        '-DFEATURE_harfbuzz:BOOL=OFF'
+        '-DFEATURE_glib:BOOL=OFF'
+        '-DFEATURE_icu:BOOL=OFF'
+        '-DFEATURE_itemmodeltester:BOOL=OFF'
+        '-DFEATURE_libjpeg:BOOL=ON'
+        '-DFEATURE_libpng:BOOL=ON'
+        '-DFEATURE_macdeployqt:BOOL=OFF'
+        '-DFEATURE_openssl:BOOL=OFF'
+        '-DFEATURE_pcre2:BOOL=ON'
+        '-DFEATURE_pdf:BOOL=OFF'
+        '-DFEATURE_printsupport:BOOL=OFF'
+        '-DFEATURE_printer:BOOL=OFF'
+        '-DFEATURE_printdialog:BOOL=OFF'
+        '-DFEATURE_printpreviewdialog:BOOL=OFF'
+        '-DFEATURE_printpreviewwidget:BOOL=OFF'
+        '-DFEATURE_qmake:BOOL=OFF'
+        '-DFEATURE_schannel:BOOL=ON'
+        '-DFEATURE_sql:BOOL=OFF'
+        '-DFEATURE_system_doubleconversion:BOOL=OFF'
+        '-DFEATURE_system_libjpeg:BOOL=OFF'
+        '-DFEATURE_system_libpng:BOOL=OFF'
+        '-DFEATURE_system_pcre2:BOOL=OFF'
+        '-DFEATURE_system_zlib:BOOL=OFF'
+        '-DFEATURE_testlib:BOOL=OFF'
+        '-DFEATURE_textmarkdownreader:BOOL=OFF'
+        '-DFEATURE_textmarkdownwriter:BOOL=OFF'
+        '-DFEATURE_windeployqt:BOOL=OFF'
+        '-DQT_BUILD_BENCHMARKS:BOOL=OFF'
+        '-DQT_BUILD_EXAMPLES:BOOL=OFF'
+        '-DQT_BUILD_EXAMPLES_BY_DEFAULT:BOOL=OFF'
+        '-DQT_BUILD_MANUAL_TESTS:BOOL=OFF'
+        '-DQT_BUILD_TESTS:BOOL=OFF'
+        '-DQT_BUILD_TESTS_BY_DEFAULT:BOOL=OFF'
+        '-DQT_BUILD_TOOLS_BY_DEFAULT:BOOL=OFF'
+        '-DCMAKE_IGNORE_PREFIX_PATH:PATH=C:/Strawberry/c'
+    )
+
+    $Options = ($Options -join ' ') -replace '-G Visual Studio \d+ \d+','-G Ninja' -replace "-A ${Target}",''
+
+    Log-Information "Config qtbase (${Target})"
 
     $Params = @{
-        BasePath = (Get-Location | Convert-Path)
-        BuildPath = "."
-        BuildCommand = "${BuildCommand}"
+        BasePath = ${Directory}
+        BuildPath = 'qtbase'
+        BuildCommand = "cmake -S . -B build_${Target} ${Options}"
         Target = $Target
-        HostArchitecture = $Target
     }
 
+    $Backup = @{
+        PATH = $env:PATH
+        VCPKG_ROOT = $env:VCPKG_ROOT
+    }
+    $env:PATH = "$(Resolve-Path ((Get-Command git).Source + '/../../usr/bin') | Convert-Path);$env:PATH"
+    $env:VCPKG_ROOT = ''
     Invoke-DevShell @Params
+    $Backup.GetEnumerator() | ForEach-Object { Set-Item -Path "env:\$($_.Key)" -Value $_.Value }
 }
 
 function Build {
-    Log-Information "Build (${Target})"
-    Set-Location $Path
+    Log-Information "Build qtbase (${Target})"
```

Comment:
Ah, disregard. It was clarified off-thread that building Qt from submodule source archives (where you must build qtbase first manually) is a little different from building from their Git repository (where you can configure and build from the super-repo and it will handle build order for you).